infra/feast-operator/internal/controller/services/services_types.go

```diff
@@ -172,22 +176,25 @@ type RepoConfig struct {
 
 // OfflineStoreConfig is the configuration that relates to reading from and writing to the Feast offline store.
 type OfflineStoreConfig struct {
-	Host string            `yaml:"host,omitempty"`
-	Type OfflineConfigType `yaml:"type,omitempty"`
-	Port int               `yaml:"port,omitempty"`
+	Host         string                 `yaml:"host,omitempty"`
```

Comment:
I'm wondering where these `Host` and `Port` fields come from. Sorry if I didn't notice them before.
I imagine they came from some initial design that was changed later, I think we can remove them.
We can do in a separate PR if you prefer, I posted this comment just to collect your opinions @tchughesiv @tmihalac

Reply:
Those should stay. They're needed for client repoconfig creation.